Packs/ContentManagement/IncidentFields/incidentfield-Reviewer.json

```diff
@@ -0,0 +1,34 @@
+{
+    "associatedToAll": false,
+    "associatedTypes": [
+        "Pull Request Creation"
+    ],
+    "caseInsensitive": true,
+    "cliName": "cicdreviewer",
+    "closeForm": false,
+    "content": true,
+    "description": "Github username to set as pull request reviewer.",
+    "editForm": true,
+    "group": 0,
+    "hidden": false,
+    "id": "incident_cicdreviewer",
+    "isReadOnly": false,
+    "locked": false,
+    "name": "CI/CD Pull Request Reviewer",
```

Comment:
what's the difference between this field and `Packs/ContentManagement/IncidentFields/incidentfield-PullRequestReview.json`

Reply:
CI/CD Pull Request Reviewer - its the user that review the PR
CI/CD Pull Request Review - the review for the PR for users that works on XSOAR and not familiar with github